docs/mine-hnt/data-only-guides/dragino.mdx

```diff
@@ -160,15 +160,15 @@ packet forwarder is communicating with the Helium Gateway service running on the
 Dragino and is getting replies from the service as expected. Note, this is not a
 foolproof way to confirm it is configured correctly. By default, the LoRaWAN
 Server settings are configured for the TTN EU server which will also provide
-acknowledgement. However, the absence of acknowledgement could indicate the the
+acknowledgment. However, the absence of acknowledgment could indicate the the
 LoRaWAN settings are not correct per the above.
 
 ## COMING SOON: Onboard as Data-Only Hotspot
 
 As part of Milestone 1.5 of the Light Gateway project roadmap, you will be able to
-onboard light gateways such as the Dragino to appear on the coverage map, and then,
+onboard data only hotspots such as the Dragino to appear on the coverage map, and then,
 in Milestone 2, earn HNT for data transfers (but not for Proof of Coverage). See
-[Light Gateways - BETA](https://docs.helium.com/mine-hnt/light-hotspots) for
+[Light HotSpots - BETA](/mine-hnt/light-hotspots) for
```

Comment:
Lowercase in "Hotspots"
```suggestion
[Light Hotspots - BETA](/mine-hnt/light-hotspots) for
```

Reply:
I fixed this, also the Only place in the whole tree like that! Doh!